regulations/make.py

```diff
@@ -210,7 +211,12 @@ def buildLanguage(args, language):
 
   pdfName = languageData[language]["pdf"]
 
-  html.html(language, buildDir, pdfName + ".pdf", isTranslation=isTranslation, verbose=args.verbose)
+  srcDir = "translations/" + language if isTranslation else "wca-regulations"
+
+  warnings = check_missing.check_guidelines(language, srcDir)
+  if len(warnings) > 0:
+    print warnings.encode('utf-8')
```

Comment:
I'm not quite sure about that : we have to utf8-decode the label in the guideline (here : https://github.com/viroulep/worldcubeassociation.org/blob/check_missing_guidelines/regulations/check_missing.py#L29), so from there `warnings` is a `<unicode>`.

Turns out printing to my terminal is fine because python knows it's utf8, however when piping/redirecting the output python doesn't know which encoding to use, and by default it will try to convert the `<unicode>` to bytes using ascii, so we need to tell python to use utf8.

I hope I'm explaining it clearly :p